ensemble/ctf/window_function_component.py

```diff
@@ -97,7 +97,7 @@ def values(self):
 
         xs = np.linspace(center - radius, center + radius, num_samples)
         ys = self.window_func(num_samples) * self.opacity
-        return zip(xs, ys)
+        return list(zip(xs, ys))
```

Comment:
Hmm... Probably needed, but it just makes me wonder about the downstream code (keeping in mind that I wrote most of said code... ahem).